### DISCUSSION TITLE
Flaslist don't work with mobx when update prop of obj

### DISCUSSION THREAD

**Discussion by LuonggNguyen** [infinitered/ignite#1991] [Q&A]
I update the object's properties but the flatlist data doesn't update, I have to make the interface re-render for the flatlist to update.
Anyone who has had this problem before, please help me, thanks a lot.

**Comment by jamonholmgren** [comment]
@LuonggNguyen Can you provide some code? I suspect your item components aren't wrapped in observer, but I'd like to see the code first.

**Reply by LuonggNguyen** [reply]
I used observer in component containing flatlist

![image](https://user-images.githubusercontent.com/87462814/182748434-03b25143-ea57-4f9d-a591-0cdfe532b34f.png)
![image](https://user-images.githubusercontent.com/87462814/182748194-a0a1dc8d-834c-46f9-8cbd-dd1ec56f4544.png)
